utilities/write_batch_with_index/write_batch_with_index_test.cc

```diff
@@ -1021,6 +1067,29 @@ TEST_F(WriteBatchWithIndexTest,
   iter->Next();
   ASSERT_OK(iter->status());
   ASSERT_FALSE(iter->Valid()) << "Should have reached upper_bound";
+
+  iter->SeekToLast();
+  ASSERT_OK(iter->status());
+  ASSERT_TRUE(iter->Valid());
+
+  AssertIter(iter.get(), "k07", "v07");
+  iter->Next();
+  ASSERT_OK(iter->status());
+  ASSERT_FALSE(iter->Valid()) << "Should have reached upper_bound";
+
+  // iter->SeekToLast();
+  // ASSERT_OK(iter->status());
+  // ASSERT_TRUE(iter->Valid());
+  // iter->Prev();
+  // ASSERT_OK(iter->status());
+  // ASSERT_TRUE(iter->Valid());
+
+  // AssertIter(iter.get(), "k06", "v06");
+  // iter->Next();
+  // AssertIter(iter.get(), "k07", "k07");
+  // iter->Next();
+  // ASSERT_OK(iter->status());
+  // ASSERT_FALSE(iter->Valid()) << "Should have reached upper_bound";
```

Comment:
Why are these commented out?

Reply:
@mrambacher It was a work in progress.

Just to update you on progress - The approach I took initially in this PR was not strong enough, with further tests other problems emerged. I now have many more tests and just one that is now failing. As soon as I have that test passing, I will push my new approach to this PR...